src/mscorlib/ref/mscorlib.cs

```diff
@@ -4473,6 +4473,8 @@ public partial class Dictionary<TKey, TValue> : System.Collections.Generic.IColl
         public Dictionary() { }
         public Dictionary(System.Collections.Generic.IDictionary<TKey, TValue> dictionary) { }
         public Dictionary(System.Collections.Generic.IDictionary<TKey, TValue> dictionary, System.Collections.Generic.IEqualityComparer<TKey> comparer) { }
+        public Dictionary(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TKey, TValue>> enumerable) {}
+        public Dictionary(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TKey, TValue>> enumerable, System.Collections.Generic.IEqualityComparer<TKey> comparer) {}
```

Comment:
I don't think it's necessary to update this file, though, it doesn't hurt... As long as you're adding the new APIs here, the parameter names should match.

Also, looks like the style of this file is to have a space between the `{` and `}`